spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -323,9 +323,43 @@ def get_rating_summary(app_id, platform, versionId = '')
       parse_response(r, 'data')
     end
 
+    def create_developer_response!(app_id: nil, platform: nil, review_id: nil, response: "")
```

Comment:
Can `app_id`, `platform` and `review_id` be `nil`?

According to L332, these seem to be required.
If these values are required, default arguments should not be passed.
They have to be nonnull.